src/command_modules/azure-cli-network/HISTORY.rst

```diff
@@ -3,6 +3,9 @@
 Release History
 ===============
 
+2.0.24
+++++++
+
```

Comment:
This should be part of 2.0.23 I believe, as it has not yet been released.  You don't need to add a history entry since these are just docs changes.